pensjon-brevbaker-api-model/src/test/kotlin/no/nav/pensjon/brev/api/model/maler/BrevkodeTest.kt

```diff
@@ -7,7 +7,7 @@ import kotlin.test.assertTrue
 class BrevkodeTest {
     @Test
     fun `ingen koder har lengde over 50`() {
-        assertTrue(Brevkode.AutoBrev.entries.map { it.name.length }.none { it > 50 }, "Alle brevkoder må være under 50 lange for å kunne arkiveres.")
+        assertTrue(Pesysbrevkoder.AutoBrev.entries.map { it.name.length }.none { it > 50 }, "Alle brevkoder må være under 50 lange for å kunne arkiveres.")
```

Comment:
Kan vi få en test for redigerbare brevkoder også?

Reply:
Så absolutt. Slo meg no at vi nok burde validert lengde ein plass i vår kode også.

Reply:
Der var det nokre (som heldigvis ikkje er skrudd på i prod enno) som feila, ja. Kjem med ein ny PR på det.